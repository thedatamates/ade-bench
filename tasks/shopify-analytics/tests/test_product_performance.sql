-- Test product performance metrics for top 5 products by revenue
SELECT 
    product_id,
    product_title,
    total_sales_revenue,
    total_units_sold,
    refund_amount,
    refund_count,
    discount_amount,
    tax_amount,
    net_revenue,
    avg_selling_price
FROM product_performance
ORDER BY total_sales_revenue DESC
LIMIT 5;