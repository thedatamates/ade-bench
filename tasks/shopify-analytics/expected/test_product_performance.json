[
  {
    "product_id": 1,
    "product_title": "Example Product 1",
    "total_sales_revenue": 10000.00,
    "total_units_sold": 100,
    "refund_amount": 500.00,
    "refund_count": 5,
    "discount_amount": 1000.00,
    "tax_amount": 800.00,
    "net_revenue": 8300.00,
    "avg_selling_price": 100.00
  }
]